www/templates/control/main-collect-settings.html

```diff
@@ -2,8 +2,7 @@
 <ion-content style="margin: 0; padding: 0;">
 <ion-button-bar>
 
-    <button class="button button-active button-block" ng-click="saveAndReloadCollectionSettingsPopover()" style="margin: 0; color: #fff; background-color: #95BCA2; border-bottom-left-radius: 0;
-border-bottom-right-radius: 0;">Save</button>
+    <button class="button button-active button-block" ng-click="saveAndReloadCollectSettingsPopover()" style="margin: 0; color: #fff; background-color: #95BCA2; border-bottom-left-radius: 0; border-bottom-right-radius: 0;">Save</button>
```

Comment:
`saveAndReloadCollectionSettingsPopover` is correct. Please don't revert my change :)

I fixed this bug as part of the last release. Did you pull?
https://github.com/e-mission/e-mission-phone/pull/54/commits/50dec5e1c38c38943c8d03418e469a97e6584b0a